lib/curly/presenter.rb

```diff
@@ -52,6 +52,10 @@ def initialize(context, options = {})
           end
         end
 
+        if value.respond_to?(:call)
+          value = value.call
```

Comment:
Yeah, that could be a problem. Thanks for catching that – would you like to do a PR?